src/instructlab/model/train.py

```diff
@@ -1141,17 +1160,26 @@ def _run_phased_training(
             journal.journal.train_2 = phase_model
         journal.commit()
 
-        if journal.journal.eval_1 is None:
-            raise RuntimeError(
-                "Training journal field 'eval_1' cannot be None for phase 'train_2'"
-            )
+        # if journal.journal.eval_1 is None:
+        #     raise RuntimeError(
+        #         "Training journal field 'eval_1' cannot be None for phase 'train_2'"
+        #     )
+
+        # NOTE:
+        # this is a recent change, implemented to ignore MMLU. We just look at the checkpoints
+        # from the phase 1 training and take the most recent one.
+        phase1_checkpoints = sorted(
```

Comment:
Are we sure this will always sort directories into the order of most recent -> least recently used? After quickly looking, I haven't seen anything that says otherwise. I'm just curious if there are any docs that explicitly say this is the best way to go.

Reply:
Alternatively can we just take the 7th checkpoint directory and not do any sorting?